resources/assets/components/CampaignOverview/index.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+
+import CampaignTable from '../CampaignTable';
+
+class CampaignOverview extends React.Component {
+  render() {
+    const causeData = this.props;
+
+    const causeTables = Object.keys(causeData).map((cause, index) => {
```

Comment:
This is some solid vanilla JS. 💪 We've also used Lodash's [`map`](https://lodash.com/docs/#map) helper for mapping over objects elsewhere, which is a little bit easier to read at first glance.